scripts/__fixtures__/false-positives.tsx

```
@@ -0,0 +1,77 @@
+      <div className={`shimmer ${styles.button}`} />
+      <div className={`btn btn-primary ${styles.pluginStoreBtn}`} />
+      <i className="fi fi-rr-home me-2" />
+      <i className="fa fa-times" />
+
+      {/* Date formats - should be skipped */}
+      <input placeholder="YYYY-MM-DD" />
+      <div>{`YYYY-MM-DDTHH:mm:ss.SSS[Z]`}</div>
+      <div>{`HH:mm:ss`}</div>
+
+      {/* Technical attributes - should be skipped */}
+      <div data-testid="my-test" role="button" aria-hidden="true" />
```

Biome (2.1.2)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)